File: java2csharp/java/Samsung/RichNotifications/DeviceStateReceiver.java
package com.samsung.android.richnotification.sample;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.util.Log;
import android.widget.Toast;

public class DeviceStateReceiver extends BroadcastReceiver {

    private static final String TAG = DeviceStateReceiver.class.getSimpleName();
    
    @Override
    public void onReceive(Context context, Intent intent) {
        boolean isConnected = intent.getBooleanExtra("isConnected", false);
        
        Log.d(TAG, "Connected : " + isConnected);
        
        Toast.makeText(context, "Connected : " + isConnected, Toast.LENGTH_LONG).show();
        
    }

}

File: java2csharp/csharp/Samsung/RichNotifications/DeviceStateReceiver.cs
namespace com.samsung.android.richnotification.sample
{

	using BroadcastReceiver = android.content.BroadcastReceiver;
	using Context = android.content.Context;
	using Intent = android.content.Intent;
	using Log = android.util.Log;
	using Toast = android.widget.Toast;

	public class DeviceStateReceiver : BroadcastReceiver
	{

		private static readonly string TAG = typeof(DeviceStateReceiver).Name;

		public override void onReceive(Context context, Intent intent)
		{
			bool isConnected = intent.getBooleanExtra("isConnected", false);

			Log.d(TAG, "Connected : " + isConnected);

			Toast.makeTextuniquetempvar.show();

		}

	}

}